pkgs/development/python-modules/trianglesolver/default.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+}:
+buildPythonPackage rec {
+  pname = "trianglesolver";
+  version = "1.2";
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-SvGKreV51cDWQ4mz5lrq8Gz/JjGXYszYWeMmhVmnauo=";
+  };
+
+  pyproject = true;
+  build-system = [ setuptools ];
```

just a nit in order and blank
```suggestion
  pname = "trianglesolver";
  version = "1.2";
  pyproject = true;

  src = fetchPypi {
    inherit pname version;
    hash = "sha256-SvGKreV51cDWQ4mz5lrq8Gz/JjGXYszYWeMmhVmnauo=";
  };

  build-system = [ setuptools ];
```